misk/src/main/kotlin/misk/MiskServiceModule.kt

```diff
@@ -184,8 +182,21 @@ class MiskCommonServiceModule : KAbstractModule() {
 
 }
 
-internal data class DependencyEdge(val service: Key<*>, val dependency: Key<*>)
-internal data class EnhancementEdge(val service: Key<*>, val enhancement: Key<*>)
+/**
+ * Normal dependency edge in the service dependency graph.
+ *
+ * @param service Identifier for the dependent service.
+ * @param dependsOn Identifier for the service that [service] depends on.
+ */
+internal data class DependencyEdge(val service: Key<*>, val dependsOn: Key<*>)
+
+/**
+ * Enhancement edge in the service dependency graph.
+ *
+ * @param toBeEnhanced The key of the service to be enhanced.
+ * @param enhancement The key of the service that enhances [toBeEnhanced].
```

Comment:
The parameter docs here might be a bit redundant, but I think its necessary to be very explicit on this, considering I wrote this API and then was subsequently confused by it 😓

Reply:
Nevermind 🤐